database/src/main/resources/db/migration/R__GTFS_query.sql

```diff
@@ -532,6 +532,8 @@ SELECT COALESCE(rh."route-short-name",'') AS "route-short-name",
   -- Join unnested per route hashes
   JOIN LATERAL unnest(dh."route-hashes") rh ON TRUE
  WHERE rh.hash IS NOT NULL
+  AND rh."route-hash-id" IS NOT NULL
```

Comment:
Just a note. River formatting would be nice. (you can refactor this in another pr)